Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -46,6 +54,18 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
     let rewards = project
       .map { $0.rewards }
 
+    self.title = configData
+      .map(third)
+      .combineLatest(with: self.viewDidLoadProperty.signal.ignoreValues())
+      .map(first)
+      .map(titleForContext)
+
+    self.backedRewardIndexPath = Signal.combineLatest(project, rewards)
+      .takeWhen(self.viewDidLayoutSubviewsProperty.signal.ignoreValues())
```

Comment:
Using `viewWillAppearProperty` here didn't do anything, so `viewDidLayoutSubviews` was the way to go.

Reply:
Interesting 🤔 I believe it needs a run loop to render the cells. We could probably go ahead with this but for interest sake, if you used `viewWillAppear` and added `.ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)` which is a delay of `0` and just puts the emission on the next run loop, that would also work.